en/flight_modes_mc/mission.md

```diff
@@ -0,0 +1,220 @@
+# Mission Mode (Multicopter)
+
+[<img src="../../assets/site/position_fixed.svg" title="Global position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
+
+_Mission mode_ causes the vehicle to execute a predefined autonomous [mission](../flying/missions.md) (flight plan) that has been uploaded to the flight controller.
+The mission is typically created and uploaded with a Ground Control Station (GCS) application like [QGroundControl](https://docs.qgroundcontrol.com/master/en/) (QGC).
+
+:::note
+
+- This mode requires a global 3d position estimate (from GPS or inferred from a [local position](../ros/external_position_estimation.md#enabling-auto-modes-with-a-local-position)).
+- The vehicle must be armed before this mode can be engaged.
+- This mode is automatic - no user intervention is _required_ to control the vehicle.
+- RC control switches can be used to change flight modes on any vehicle.
+- RC stick movement will [by default](#COM_RC_OVERRIDE) change the vehicle to [Position mode](../flight_modes_mc/position.md) unless handling a critical battery failsafe.
```

Comment:
Is this true also for FW?

Reply:
No, override is only possible for MC and VTOL in hover mode.

Reply:
```suggestion
- RC stick movement will [by default](#COM_RC_OVERRIDE) change the vehicle to [Position mode](../flight_modes_mc/position.md) unless handling a critical battery failsafe.
  This is true for multicopters and VTOL in MC mode.
```